backend/src/Installer.ts

```diff
@@ -126,14 +151,23 @@ export default class Installer {
    * Download the binary, move it to the PATH on the user's
    * system and then make it writable.
    */
-  install(tempDir: string, cb?: ProgressCallback) {
+  install(cb?: ProgressCallback) {
```

Comment:
This method used to download and move the files... I would think since the files are already local that we would just be selecting and moving the files and not need a tmp dir? Probably will have to refactor more and update the installBinaries.ts file as well.